packages/utilities/src/focus.ts

```diff
@@ -191,7 +204,8 @@ export function getNextElement(
   suppressParentTraversal?: boolean,
   suppressChildTraversal?: boolean,
   includeElementsInFocusZones?: boolean,
-  allowFocusRoot?: boolean): HTMLElement | null {
+  allowFocusRoot?: boolean,
+  tabbable?: boolean): HTMLElement | null {
```

Comment:
what does tabbable mean?

Reply:
Naming and parameter concept are copied from existing sibling method getPreviousElement.  'tabbable' is passed along as the 2nd parameter to isElementTabbable which is really checking if an element is focusable, except that when the 2nd param==true there is an additional check that tabIndex != -1 meaning you can use the tab key to get to it.  The existing public naming standards in this file are terrible.

Reply:
Ah yeah, that right, thanks for reminding me